app/src/main/java/ar/scacchipa/twittercloneapp/fragment/FragmentAuthWebDialog.kt

```diff
@@ -29,7 +29,7 @@ class FragmentAuthWebDialog : Fragment() {
 
         binding = FragmentAuthWebDialogLayoutBinding.inflate(inflater)
 
-        viewModel.userAccessToken.observe(viewLifecycleOwner) {
+        viewModel.addTokenObserver(viewLifecycleOwner) {
             when {
                 it.error == Constants.ERROR_CANCELLED_AUTH -> {
                     findNavController().navigateUp()
```

Comment:
to validate if the token is "" better to use a function like it.isBlank or isEmpty, choose whatever fits your needs

Reply:
Also remember to remove the logs before pushing

Reply:
"" was replaced and the Log were deleted.